### PR TITLE
[PART 2] Improvement/5242 migrate junit4 to junit5 v2

### PR DESCRIPTION
**Pull request by IvanChupin**
This PR should be merged right after this one: https://github.com/telstra/open-kilda/pull/5260

- [ ] stats-topology
- [ ] base-topology
- [ ] blue-green
- [ ] connecteddevices-topology
- [ ] kilda-persistence-tinkerpop
- [ ] isllatency-topology
- [ ] opentsdb-topology
- [ ] ping-topology
- [ ] flowhs-topology
- [ ] flowmonitoring-topology
- [ ] nbworker-topology
- [ ] kilda-pce
- [x] floodlight-service
- [ ] floodlightrouter-topology
- [ ] grpc-speaker
- [ ] kilda-configuration
- [ ] kilda-model
- [ ] kilda-persistence-api
- [ ] network-topology
- [ ] northbound-service
- [ ] testing
- [ ] reroute-topology
- [ ] rule-manager
- [ ] server42
- [ ] swmanager-topology

Here we have the Junit4->Junit5 migration for  floodlight-service module.

closes: #5242

### CODE REVIEW

File: src-java/floodlight-service/floodlight-modules/src/test/java/org/openkilda/floodlight/switchmanager/SwitchTrackingServiceTest.java [hunk 1]
```diff
@@ -81,7 +81,7 @@
 import java.util.Map;
 import java.util.Set;
 
-@Ignore("nmarchenko must be fixed after mfl merging")
+@Disabled("nmarchenko must be fixed after mfl merging")
```
Comment (dmitrii-beliakov) [inline src-java/floodlight-service/floodlight-modules/src/test/java/org/openkilda/floodlight/switchmanager/SwitchTrackingServiceTest.java:84]: I created an issue for it

File: src-java/floodlight-service/floodlight-modules/src/test/java/org/openkilda/floodlight/utils/CorrelationContextTest.java [hunk 2]
```diff
@@ -26,20 +26,20 @@ public void shouldRestoreCorrelationIdOnClose() {
...
-            assertThat(CorrelationContext.getId(), not(equalTo(before)));
+            MatcherAssert.assertThat(CorrelationContext.getId(), not(equalTo(before)));
         }
 
-        assertThat(CorrelationContext.getId(), equalTo(before));
+        MatcherAssert.assertThat(CorrelationContext.getId(), equalTo(before));
     }
 
-    @Ignore("Fix applying aspects to test classes")
+    @Disabled("Fix applying aspects to test classes")
```
Comment (dmitrii-beliakov) [inline src-java/floodlight-service/floodlight-modules/src/test/java/org/openkilda/floodlight/utils/CorrelationContextTest.java:35]: I created an issue for it

File: src-java/flowhs-topology/flowhs-storm-topology/src/test/java/org/openkilda/wfm/topology/flowhs/service/FlowRerouteServiceTest.java [hunk 3]
```diff
@@ -336,7 +332,7 @@ public void shouldFailRerouteOnTimeoutDuringValidation()
                 any(String.class));
     }
 
-    @Ignore("FIXME: need to replace mocking of updateStatus with another approach")
+    @Disabled("FIXME: need to replace mocking of updateStatus with another approach")
     @Test
     public void shouldFailRerouteOnSwapPathsError()
```
Comment (dmitrii-beliakov) [inline src-java/flowhs-topology/flowhs-storm-topology/src/test/java/org/openkilda/wfm/topology/flowhs/service/FlowRerouteServiceTest.java:337]: I created an issue for it

File: src-java/flowhs-topology/flowhs-storm-topology/src/test/java/org/openkilda/wfm/topology/flowhs/service/FlowUpdateServiceTest.java [hunk 4]
```diff
@@ -352,7 +348,7 @@ public void shouldFailUpdateOnTimeoutDuringValidation()
         verifyNoPathReplace(origin, result);
     }
 
-    @Ignore("FIXME: need to replace mocking of updateStatus with another approach")
+    @Disabled("FIXME: need to replace mocking of updateStatus with another approach")
```
Comment (dmitrii-beliakov) [inline src-java/flowhs-topology/flowhs-storm-topology/src/test/java/org/openkilda/wfm/topology/flowhs/service/FlowUpdateServiceTest.java:351]: I created an issue for it

File: src-java/flowhs-topology/flowhs-storm-topology/src/test/java/org/openkilda/wfm/topology/flowhs/service/yflow/YFlowCreateServiceTest.java [hunk 5]
```diff
@@ -286,7 +286,7 @@ public void shouldFailOnTimeoutDuringMeterInstallation()
         verifyYFlowIsAbsent(request.getYFlowId());
     }
 
-    @Ignore("TODO: implement meter validation")
+    @Disabled("TODO: implement meter validation")
```
Comment (dmitrii-beliakov) [inline src-java/flowhs-topology/flowhs-storm-topology/src/test/java/org/openkilda/wfm/topology/flowhs/service/yflow/YFlowCreateServiceTest.java:289]: I created an issue for this class

**Review by dmitrii-beliakov — APPROVED** [review]
There are several EOL missing in the end of file. I didn't comment on lines I already commented in  v1 PR. Otherwise all is good!

File: src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/flow/resources/CookiePoolTest.java [hunk 6]
```diff
@@ -58,12 +59,14 @@ public void cookiePool() {
         });
     }
 
-    @Test(expected = ResourceNotAvailableException.class)
+    @Test()
```
[the remark this answers is not in the document]
Comment (IvanChupin) [reply src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/flow/resources/CookiePoolTest.java:62]: fixed

File: src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/flow/resources/FlowResourcesManagerTest.java [hunk 7]
```diff
@@ -233,15 +234,17 @@ public void deallocatePathResourcesTest() throws Exception {
...
-    @Test(expected = ResourceAllocationException.class)
-    public void shouldThrowExceptionOnAllocationFailed() throws ResourceAllocationException {
-        transactionManager.doInTransaction(() -> {
-            FlowResourcesManager spy = Mockito.spy(resourcesManager);
-            Mockito.doThrow(ConstraintViolationException.class)
-                    .when(spy).allocateResources(any(), any(), any());
-
-            Flow flow = convertFlow(firstFlow);
-            spy.allocateFlowResources(flow);
+    @Test()
```
[the remark this answers is not in the document]
Comment (IvanChupin) [reply src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/flow/resources/FlowResourcesManagerTest.java]: fixed

File: src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/flow/resources/vxlan/VxlanPoolTest.java [hunk 8]
```diff
@@ -72,14 +73,14 @@ public void vxlanIdPool() {
     }
 
 
-    @Test(expected = ResourceNotAvailableException.class)
+    @Test()
```
[the remark this answers is not in the document]
Comment (IvanChupin) [reply src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/flow/resources/vxlan/VxlanPoolTest.java]: fixed

File: src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/service/IntersectionComputerTest.java [hunk 9]
```diff
@@ -314,41 +314,42 @@ public void calculateSharedPathAsFullPathTest() {
...
+                    .segments(Lists.newArrayList(
+                            buildPathSegment(NEW_PATH_ID, SWITCH_ID_A, SWITCH_ID_B, 1, 1),
+                            buildPathSegment(NEW_PATH_ID, SWITCH_ID_B, SWITCH_ID_C, 2, 2)))
+                    .build();
+            IntersectionComputer.calculatePathIntersectionFromSource(Collections.singletonList(firstPath));
+        });
     }
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test()
```
[the remark this answers is not in the document]
Comment (IvanChupin) [reply src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/service/IntersectionComputerTest.java]: fixed

**Review by pablomuri — APPROVED** [review]

**Pull request merged**